src/webgl/light.js

```diff
@@ -592,13 +592,64 @@ p5.prototype.pointLight = function(v1, v2, v3, x, y, z) {
  * @alt
  * light with slider having a slider for varying roughness
  */
-p5.prototype.imageLight = function(img){
+p5.prototype.imageLight = function (img) {
   // activeImageLight property is checked by _setFillUniforms
   // for sending uniforms to the fillshader
   this._renderer.activeImageLight = img;
   this._renderer._enableLighting = true;
 };
 
+/**
+ * Creates a Panorama with given image.
+ *
+ *
+ * `panorama(img)` is a method designed to transform a standard
+ * image into a 360-degree view. It operates on the concept
+ * of sphere mapping, where the image is manipulated to
+ * resemble a sphere by adjusting camera angles. Utilizing
+ * this method, users can obtain a complete 360-degree view
+ * of a scene.
+ *
+ * Using Panorama is straightforward. Similar to calling a
```

Comment:
This section where we describe it as being similar to background is actually maybe a good thing to start this documentation with instead of ending it with it, what do you think?